openlibrary/macros/ShareModal.html

```diff
@@ -42,13 +42,13 @@
                 <a class="embed-work-btn" title="$_('Embed this book in your website')" data-ol-link-track="Share|Embed"
                   onclick="prompt('Copy embed code to clipboard:', '$iframe_html');">
                   <img
-                    alt="Embed icon"
+                    alt="$_('Embed icon')"
                     class="share-link"
                     src="$static_url('images/embed.png')"
                     loading="lazy"
                   >
                 </a>
-                <div class="share-source">$_('Embed')</div>
+                <div class="share-source">$_("Embed")</div>
```

Comment:
Why was this change made?

Reply:
My understanding is that we're using double quotes for i18n syntax for innerHTML.

Reply:
Oh this was my bad! There was a miscommunication ; I confirmed to Rebecca that we prefer double quotes, but I didn't clarify that I meant that we should have all _new_ i18n strings using double quotes, not that we should edit existing ones to use double quotes! But the contribution to i18n in this PR was so valuable that I didn't feel it was a good use of @merwhite11 's time to change them back, so didn't mention it. My bad!